pkgs/data/icons/super-tiny-icons/default.nix

```
@@ -0,0 +1,36 @@
+{ pkgs, lib, stdenvNoCC, fetchFromGitHub, }:
+
+stdenvNoCC.mkDerivation {
+  pname = "super-tiny-icons";
+  version = "unstable-2022-11-07";
+
+  src = fetchFromGitHub {
+    owner = "edent";
+    repo = "SuperTinyIcons";
+    rev = "b4d5a3be04c99ec0a309ac9e0d0b21207c237c7d";
+    sha256 = "670ndAuBrZBr7YOTQm2zHJfpiBC56aPk+ZKMneREEoI=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/icons/SuperTinyIcons
+    for icondir in $(find $src/images -type d); do
+      cp -r $icondir $out/share/icons/SuperTinyIcons/
+    done
```

In general I dislike having bash split the output of find.
Perhaps something like this?
```suggestion
    find $src/images -type d -exec cp -r {} $out/share/icons/SuperTinyIcons/ \;
```